### DISCUSSION TITLE
instalacion de ipywidgets

### DISCUSSION THREAD

**Discussion by EinarDvls** [LaunchX-InnovaccionVirtual/CursoIntroPython#46] [Q&A]
hola a todos me podrian asesorar con un problema que tengo, la verdad al poner el comando del modulo 0 "pip install ipywidgets" me marca este tipo de errores y ya estuve buscando y nada me funciona

![Captura](https://user-images.githubusercontent.com/99060661/153281448-07b99749-2aa0-483e-84ee-682aa6f1c866.PNG)
![1](https://user-images.githubusercontent.com/99060661/153281450-f532a943-b47b-44c7-b577-de82b8896df4.PNG)

**Comment by MarcosSilvaG** [comment]
Podrías proporcionar un poco mas de información?
Como dónde estás ejecutando el código o cómo lo estás haciendo, para poder tener más idea de lo que pueda estar pasando :D

**Reply by EinarDvls** [reply]
lo he tratado de instalar desde vs code o desde la terminal y en las dos opciones me da el mismo error
![error](https://user-images.githubusercontent.com/99060661/153292210-fefcbd5a-19c8-47bf-8386-3e7dd0d2b37e.PNG)

**Reply by MSuarez0** [reply]
te sugiero que vayas ala carpeta de python y trates de ejecutarlo desde hay
y ejecutes el comando python3 pip install ipywidgets
o 
pip3 install ipywidgets

**Comment by KevinChavarin** [comment]
Instala python 3.10 desde la microsoft store, el problema debería quedar solucionado

**Reply by EinarDvls** [reply]
Mi computadora no soporta el Python 3.10 por qué tengo una con Windows 7 Ultimate de 32 bits

**Reply by marcopasos** [reply]
Marco Pasos Hola : tengo el mismo problema tengo Windows 7 Ultimate de 32 bits. Pudiste solucionar?
